include/keyence/impl/high_speed_defs.h

```diff
@@ -3,9 +3,28 @@
 
 #include <stdint.h>
 
+#define KEYENCE_FUNDAMENTAL_LENGTH_UNIT (0.01 * 1.0e-6)
+// The integer encoded values representing 'out of range'
+// received from the sensor
+#define KEYENCE_INFINITE_DISTANCE_VALUE -524288
```

Comment:
And @shaun-edwards: these values are sort-of logical: they all encode out-of-range or invalid data points. And closer points (as in: closer to the sensor) have larger values, so 'negative infinity' (what this essentially is) is used to encode errors.